regulations/static/regulations/css/scss/comment/_comment-read.scss

```diff
@@ -106,10 +106,10 @@ Preamble Read Mode
       height: 10px;
       width: 10px;
       vertical-align: middle;
-      background-color: $gray_lighter;
+      background-color: $panel_background_color;
```

Comment:
Good call, reverted this to the darker color so it would stand out from the white background.